host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -1448,7 +1594,30 @@ static MMAL_STATUS_T create_camera_component(RASPIVID_STATE *state)
    // Set the encode format on the video  port
 
    format = video_port->format;
-   format->encoding_variant = MMAL_ENCODING_I420;
+
+   if (state->raw_output)
+   {
+      switch (state->raw_output_fmt)
```

Comment:
Thank you for the information! I'll try to configure the splitter to pass ENCODING_OPAQUE data from camera to encoder and to do format conversion on output port.